pkgs/build-support/ocaml/dune.nix

```diff
@@ -3,7 +3,7 @@
 { pname, version, nativeBuildInputs ? [], enableParallelBuilding ? true, ... }@args:
 
 let Dune =
-  let dune-version = args . duneVersion or (if args.useDune2 or true then "2" else "1"); in
+  let dune-version = args . duneVersion or (if args.useDune2 or true then "3" else "1"); in
```

Comment:
Hmm this doesn't seem right, why did you change it?

Reply:
I think ideally at this point we should remove `useDune2` entirely.

Reply:
I should mark this as draft because I'm playing around a bit with moving things over to dune 3...